Sources/Components/Style/Legacy/UIColor+Extensions.swift

```diff
@@ -1,42 +1,38 @@
 import UIKit
 
-/// Color definitions from Wikimedia Style Guide
 public extension UIColor {
 
-    static let gray800 = UIColor(0x101418)
-    static let gray700 = UIColor(0x202122)
-    static let gray675 = UIColor(0x27292D)
-    static let gray650 = UIColor(0x2E3136)
-    static let gray600 = UIColor(0x54595D)
-    static let gray500 = UIColor(0x72777D)
-    static let gray400 = UIColor(0xA2A9B1)
-    static let gray300 = UIColor(0xC8CCD1)
-    static let gray200 = UIColor(0xEAECF0)
-    static let gray150 = UIColor(0xEEF2FB)
-    static let gray100 = UIColor(0xF8F9FA)
-    static let blue700 = UIColor(0x2A4B8D)
-    static let blue600 = UIColor(0x3366CC)
-    static let blue300 = UIColor(0x6699FF)
-    static let blue100 = UIColor(0xEAF3FF)
-    static let red700 = UIColor(0xB32424)
-    static let red600 = UIColor(0xDD3333)
-    static let red100 = UIColor(0xFEE7E6)
-    static let green600 = UIColor(0x00AF89)
-    static let green100 = UIColor(0xD5FDF4)
-    static let yellow600 = UIColor(0xFFCC33)
-    static let beige400 = UIColor(0xE1DAD1)
-    static let beige300 = UIColor(0xF0E6D6)
-    static let beige100 = UIColor(0xF8F1E3)
-    static let taupe600 = UIColor(0x646059)
-    static let taupe200 = UIColor(0xCBC8C1)
-    static let purple600 = UIColor(0x6b4ba1)
-    static let orange600 = UIColor(0xFF9500)
-    static let darkSearchFieldBackground = UIColor(0x8E8E93, alpha: 0.12)
-    static let lightSearchFieldBackground = UIColor(0xFFFFFF, alpha: 0.15)
+	static let gray800 = UIColor(0x101418)
```

Comment:
Thanks for trying to clean these out. We should be able to once we reference WKColors directly in the client app's Theme.swift file (I think the patterns were still being established at that time). But that's a bigger change that is best left to a separate PR.